src/ServerExtension.php

```diff
@@ -89,7 +89,7 @@ public function __construct(private $extension)
 
                 public function notify(TestFinishedEvent $event): void
                 {
-                    $this->extension->executeAfterTest();
+                    $this->extension->executeAfterTest($event->test()->name(), (float) $event->telemetryInfo()->time()->seconds());
```

Comment:
I'm not convinced the $time should be in seconds, especially when the value should be a float, but the value isn't used in the legacy extension anyway, so this effectively mutes the error.